### DISCUSSION TITLE
Multiple domains on one license?

### DISCUSSION THREAD

**Discussion by cmaxw** [avo-hq/avo#3247] [General]
I'm looking at building out a SaaS where someone could set up their own version of my application under their domain...

I've considered doing a separate deployment to my own servers when someone signs up and I've also looked at multitenant applications where one codebase serves a variety of domains from the same deployed codebase.

I'm looking for clarity on the licensing agreement.

Would I have to host all the avo code under my domain and then run everything else on the other domain provided by my customer? Or can I use one license to allow them to operate the admin under their domain?

Thanks,
Chuck

**Comment by adrianthedev** [comment]
Hey @cmaxw.

Apologies for the delay. It's conference season (euruko, friendly.rb, and rails world) and I got distracted for a bit.

I'm going to try and answer your question.

The way you organize the app is up to you. You can have one app and serve multiple domains from it or you can have multiple instances of the **same app** deployed.

I guess that the first scenario is a bit more manageable for you.

Avo's multi-URL licensing is a custom one and we need to hop on a call for that.
It depends on how many URLs you need.

I'll send you an email to set up a call.
